roles/shiftstack/defaults/main.yml

```diff
@@ -20,12 +20,14 @@
 cifmw_shiftstack_artifacts_dir: "{{ cifmw_shiftstack_basedir }}/artifacts"
 cifmw_shiftstack_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}/tests/shiftstack"
 cifmw_shiftstack_client_pod_image: "quay.io/shiftstack-qe/shiftstack-client:latest"
-cifmw_shiftstack_client_pod_name: "shiftstackclient"
+cifmw_shiftstack_client_pod_name: "shiftstackclient-{{ cifmw_shiftstack_project_name }}"
 cifmw_shiftstack_client_pod_namespace: "openstack"
+cifmw_shiftstack_cluster_name: "shiftstack"
```

Comment:
I think it's the other way around. The cluster name is 'ostest' and the project name is 'shiftstack'